homeassistant/helpers/update_coordinator.py

```diff
@@ -84,9 +84,18 @@ def __init__(
         self.update_interval = update_interval
         self._shutdown_requested = False
         if config_entry is UNDEFINED:
+            # late import to avoid circular imports
+            from . import frame  # pylint: disable=import-outside-toplevel
+
+            frame.report_usage(
+                "rely on the ContextVar, but should passing the config entry explicitly - see "
```

Comment:
```suggestion
                "relies on ContextVar, but should pass the config entry explicitly - see "
```

Reply:
to avoid another full-ci run, I'll adjust the wording, as soon as the blog post is ready